README.md

```diff
@@ -4,19 +4,21 @@ Benchmark `nargo prove` times of Noir programs spanning across different constra
 ## Results
 
 Existing results were gathered using:
```

Comment:
Adding a v0.26.0 results section instead of overwriting the existing v0.21.0 results might be better, as they don't fully cover everything that was benchmarked in v0.21.0. 

```suggestion
## v0.26.0 Results

Results gathered with:
```